Makefile

```diff
@@ -17,7 +17,9 @@ docs:
 	$(MAKE) -C doc
 	$(MAKE) -C man
 
-manpage:
+# "src" is a prerequisite to prevent parallel build errors.
+# manpage builds currently require a pre-built "unison" binary.
+manpage: src
```

Comment:
This fix seems fine - thanks!

Isn't it also a bug for the man makefile to reach back to src, rather than just documenting that it may only be invoked when src is up-to-date?

Reply:
It's not a bug like #774 (parallel make).

Should it be like that? Long term, probably no. With current makefiles and build setup, I think it's ok and works well.

Reply:
Fair enough, and no need to couple lesser things with fixes for real bugs.